.buildkite/pipeline.yml

```diff
@@ -1,6 +1,18 @@
 steps:
   - command: .buildkite/scripts/run_autoformat.py
     label: "autoformat"
+  - label: "Internal model tool: check compatibility"
```

Comment:
You might want to run this on branches as well as in `main` (if that's the intention - i'm guessing it is from the commented out `branches: "main"`). It would help to know pre-merge if the branch is not deployable.

Reply:
You're right about the assumption, but I wasn't sure, I'll leave it in and if it is hassle-some we can work it out then.